src/api/app/controllers/webui/package_controller.rb

```diff
@@ -440,4 +445,13 @@ def get_diff(project, package, options = {})
     end
     true
   end
+
+  def report_bug_or_bugzilla_url
+    return @package.report_bug_url if feature_enabled?('foster_collaboration') && @package.report_bug_url.present?
+
+    bugowners_mail = (@package.bugowner_emails + @project.bugowner_emails).uniq
+    return helpers.bugzilla_url(bugowners_mail, "#{@project.name}/#{@package.name}: Bug") if bugowners_mail.present? && ::Configuration.bugzilla_url.present?
```

Comment:
I think you should move this out of this helper as the only place you use this is *right* here.

Reply:
I didn't want to duplicate the code of the `bugzilla_url` helper both for projects and packages. This could be part of a follow-up refactoring of the code.

Basically, I wanted to provide the functionallity with the changes that made sense. Further refactorings are always welcome. This pull requests already refactors some code and unvelis some legacy code. But IMHO I doesn't introduce any code to be refactored in this same pull request.